airflow/bin/cli.py

```diff
@@ -607,6 +636,10 @@ class CLIFactory(object):
         'execution_date': Arg(
             ("execution_date",), help="The execution date of the DAG",
             type=parsedate),
+        'execution_date_enhanced': Arg(
```

Comment:
Why not support this as part of `execution_date`, rather than having two different params?